### PR TITLE
Add the --typescript flag, to allow the downloaded files to be written as TS ones

### PR DESCRIPTION
**Pull request by CosmaTrix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +15,12 @@ CROWDIN_LANGUAGES
...
 Add the following scripts to `package.json`:
 ```shell
-"crowdin:download": "@mollie/crowdin-cli download",
-"crowdin:upload": "@mollie/crowdin-cli upload '<glob directory pattern>'", // i.e. mollie-crowdin upload './src/**/!(*.test).{ts,tsx}'
+"crowdin:download": "mollie-crowdin download",
+"crowdin:upload": "mollie-crowdin upload '<glob directory pattern>'", // i.e. mollie-crowdin upload './src/**/!(*.test).{ts,tsx}'
+```
+
+## Typescript
+It is possible to write the downloaded files to Typescript (.ts). to Achieve that, pass the `--typescript` flag to the download command.
```
Comment (vernondegoede) [inline README.md]:
```suggestion
It is possible to write the downloaded files to Typescript (.ts). To achieve that, pass the `--typescript` flag to the download command.
```

File: package.json [hunk 2]
```diff
@@ -68,6 +68,8 @@
     "typescript": "^3.7.3"
   },
   "contributors": [
-    "Angelo Abdoelsamath <angelo.abdoelsamath@mollie.com>"
+    "Angelo Abdoelsamath <angelo.abdoelsamath@mollie.com>",
```
Comment (vernondegoede) [inline package.json]: Do you need to merge `master` into this branch?

File: README.md (hunk 1 above)
Comment (CosmaTrix) [reply README.md]: Well spotted! Thanks! 🙏

File: package.json (hunk 2 above)
Comment (CosmaTrix) [reply package.json]: It's merged now.

**Pull request merged**